package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>nanoeigenpy</name>
+  <version>0.0.0</version>
+  <description>A support library for bindings between Eigen in C++ and Python, based on nanobind</description>
+  <maintainer email="wilson.jallet@polytechnique.org">Wilson Jallet</maintainer>
+  <maintainer email="lucas.haubert@ens-paris-saclay.fr">Lucas Haubert</maintainer>
+  <author>Wilson Jallet</author>
+  <author>Lucas Haubert</author>
+  <license>BSD</license>
+
+  <url type="website">https://github.com/Simple-Robotics/nanoeigenpy</url>
+
+  <build_depend>git</build_depend>
+  <build_depend>doxygen</build_depend>
```

Comment:
Do you depend on doxygen?

Reply:
For the docs, just like eigenpy did (this comes from eigenpy's package.xml)